cdap-data-fabric/src/test/java/co/cask/cdap/data2/datafabric/dataset/RemoteDatasetFrameworkTest.java

```diff
@@ -155,8 +154,13 @@ public void onChange(ServiceDiscovered serviceDiscovered) {
 
     startLatch.await(5, TimeUnit.SECONDS);
 
+    // the framework.delete looks up namespace config through namespaceadmin add the meta there too.
+    NamespaceMeta systemMeta = new NamespaceMeta.Builder().setName(Id.Namespace.SYSTEM).build();
```

Comment:
the namespaceadmin here is just in-mem one so we need to create it explicitly. binding the default one needs a lot of dependencies which we dont have here.